.gitmodules

```diff
@@ -31,9 +31,6 @@
 [submodule "cmake/external/mimalloc"]
 	path = cmake/external/mimalloc
 	url = https://github.com/microsoft/mimalloc.git
-[submodule "cmake/external/cub"]
```

Comment:
Can all references to cub be deleted too ? For example, it should still be referenced in `ThirdpartyNotices.txt` right ?

Reply:
Thanks for pointing out. I just updated ThirdpartyNotices.txt. If there is any other remaining, please let me know.

Reply:
That's all jumps to my mind as well.